pages/[user]/book.tsx

```diff
@@ -137,7 +141,7 @@ export default function Book(props: any): JSX.Element {
     book();
   };
 
-  return (
+  return isReady && (
```

Comment:
Codacy has a fix for the issue: [Insert `(⏎····`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
  return (
```